src/playground/Widgets/ProgressSection/index.tsx

```diff
@@ -11,6 +11,7 @@ interface ProgressSectionProps extends Omit<CategoryBarProps, "data"> {
   unit?: string
   primaryLabel?: string
   secondaryLabel?: string
+  bottomComment?: string
```

Comment:
What if we name it something else? The position might change, so it's better not to imply that in the name. I'm not thrilled with just `comment` as a name, but it could work if we don't come up with a better one.